vast/integration/reference/pivot/step_06.ref

```diff
@@ -1,12 +1,12 @@
 #empty_field	(empty)
-#fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	proto	service	duration	orig_bytes	resp_bytes	conn_state	local_orig	local_resp	missed_bytes	history	orig_pkts	orig_ip_bytes	resp_pkts	resp_ip_bytes	tunnel_parents	community_id
+#fields	_path	_write_ts	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	proto	service	duration	orig_bytes	resp_bytes	conn_state	local_orig	local_resp	missed_bytes	history	orig_pkts	orig_ip_bytes	resp_pkts	resp_ip_bytes	tunnel_parents	community_id
```

Comment:
This output is not correct. The Zeek TSV writer has the metadata embedded in the header:

- `_path` is `#path`
- `_write_ts` is not of concern and can be dropped.

No tool that relies on Zeek TSV data can use this output.

Reply:
That would be a bug in the Zeek writer then, which was not changed by this PR at all. What is it supposed to do except for taking the data it gets and rendering it exactly as it is?

Reply:
> That would be a bug in the Zeek writer then, which was not changed by this PR at all.

That might be the developer view, but that doesn't matter from a user perspective. The output is now vastly different and that's what matters. 

We had this recently: making a change that has global effects and then arguing "I haven't touched the other code" is not a valid point. We must look at this from the user perspective, not (just) the developer viewpoint.

> What is it supposed to do except for taking the data it gets and rendering it exactly as it is?

It should strip Zeek-related meta data, if the input is Zeek data. My previous comments has two bullets that describe what the user expectation is. At a high level, the expectation is that meta data gets transposed into the places where the user expects it.

That said, I can see where you coming from and I agree that this breaks a symmetry. My suggestion to appease: make this transposition a toggle with an option, e.g., `--embed-metadata`.

Reply:
I don't think logic about types based on their name should sneak into the writers for specific formats. That is an approach that is (1) hidden from the user and (2) does not scale. It is bound to bite us sometime in the future.

A writer for a given format should just write data. That is its single responsibility.

Downstream tooling that works on Zeek logs should access the fields by field name, not by column index. Which downstream tool do you know of that just uses the column index? Wouldn't you always go through `zeek-cut` first?

What we can do here is the following:
- Remove `_path` from the schema. It is only needed for Zeek JSON reader to determine which schema to use (the selector field of the JSON reader), so there isn't much use in actually putting it into the data. If users really want that, they are already able to put the field back using pipeline operators.
- Move `_write_ts` to the end of all schemas. We simply cannot remove it, as it is data and not metadata.

Reply:
I pushed a commit that makes this change. If you're fine with that then all we need is another changelog entry.